app/support/price_displayment.rb

```diff
@@ -54,18 +54,34 @@ def actual_cost?
     actual_cost.present?
   end
 
-  def wrapped_quantity
+  class QuantityDisplay < Struct.new(:value, :type)
+    include ActionView::Helpers::TagHelper
+
+    def time_quantity_html
+      content_tag :span, value, :class => 'timeinput'
+    end
+
+    def output
+      type == :timeinput ? time_quantity_html : value
+    end
+  end
+
+  def quantity_display
     if reservation.try(:actual_duration_mins) && reservation.actual_duration_mins > 0
-      content_tag :span, reservation.actual_duration_mins, :class => 'timeinput'
+      QuantityDisplay.new(reservation.actual_duration_mins, :timeinput)
```

Comment:
I'm assuming you're planning to use this in other displays as well. Otherwise, is it necessary to pass `:timeinput` in?